xml/art_migration.xml

```diff
@@ -0,0 +1,692 @@
+    </para>
+    </listitem>
+    <listitem>
+      <para>
+	Any data that is not on a disk separate from the primary root partition
+	will be lost during re-imaging. This includes &swift; objects and audit logs.
+    </para>
+    </listitem>
+    <listitem>
+      <para>
+	All &compnode;s are running supported Linux distributions:
+    </para>
+    <itemizedlist>
+      <listitem>
+	<para>
+	  &sls; 12 SP3 for SLES &compnode;s
+	 </para>
+	</listitem>
+	<listitem>
+	  <para>
+	    RHEL 7.3 for RHEL &compnode;s
+	   </para>
+	  </listitem>
+    </itemizedlist>
+    </listitem>
+    <listitem>
+      <para>
+	There are only supported input model services.
+    </para>
+    <itemizedlist>
+      <listitem>
+	<para>
+	There is no embedded HOS Ceph or VSA.
+	 </para>
+	</listitem>
+    </itemizedlist>
+    </listitem>
+    <listitem>
+      <para>
+	Only supported third-party integrations and extensions are installed.
+    </para>
+    <itemizedlist>
+      <listitem>
+	<para>
+	  There are no unsupported customized virtual environments.
+	 </para>
+	</listitem>
+    </itemizedlist>
+    </listitem>
+  </itemizedlist>
+  <procedure>
+    <title>Preparation Phase</title>
+<step>
+  <para>
+    Run validation playbook to ensure that the current cloud load and configuration
+    is compatible with the migration process:
+   <remark role="clarity">taroth 2018-02-27: what is 'validation playbook'? the name of a script?
+    then it should be tagged with &lt;literal/&gt;</remark>
+  </para>
+  <screen>(old-deployer)> git clone -b hp/prerelease/newton https://gerrit.suse.provo.cloud/hp/hlm-migration-tools>
+(old-deployer)> cd hlm-migration-tools/ansible
+(old-deployer)> ansible-playbook -i ~/scratch/ansible/next/hos/ansible/hosts/verb_hosts pre-migration-validations.yml
+  </screen>
+  <para>
+    The validation playbook performs the following checks:
+  </para>
+  <itemizedlist>
+    <listitem>
+      <para>
+       Correct HOS 5.0.1/5.0.2 installed version number?
+      <remark role="clarity">taroth 2018-02-27: not sure what this should mean:
+       does the version number HOS version number need to be either 5.0.1 or 5.0.2?
+       then rephrase to: 'Is the HOS version number either 5.0.1 or 5.0.2?'</remark>
+     </para>
+    </listitem>
+    <listitem>
+      <para>
+       C8-compatible Linux distributions on &compnode;s?
+    </para>
+    </listitem>
+    <listitem>
+     <para>
+      System clock synchronization across cloud nodes?
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      HA capacity of &contrnode;s (L3/DHCP agent)?
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      HA capacity of &contrnode;s (&swift;)?
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Data model and &o_blockstore; back-end compatibility (no VSA or Ceph)?
+     </para>
+    </listitem>
+  </itemizedlist>
+</step>
+<step>
+  <para>
+   Identify a candidate seed &contrnode; in the Primary control plane. This
```

Comment:
Primary -> primary..? I see no good reason for capitalizing this.

@ devs Can you do anything about "candidate seed control node" -- maybe explain the motivation a little bit here?